CONTRIBUTING.md

```
@@ -0,0 +1,368 @@
+
+- Use **function declarations** (not const) for React components
+- Prefer **named exports** over default exports
+- Place TypeScript interfaces at the end of files
+- Follow **micro folder structure** - break components into small, focused parts
+- Minimize 'use client' usage - prefer server components
+
+**Example:**
+
+```tsx
+export function MyComponent({ title }: MyComponentProps) {
+	return <div>{title}</div>;
```

markdownlint-cli2 (0.18.1)

Hard tabs
Column: 1

(MD010, no-hard-tabs)